doc/rst/meta/static/style.css

```diff
@@ -15,3 +15,7 @@
 /* Inlined code is black (instead of default red) */
 .pre{ color: #000}
 
+/* Override the default padding added by the RTD theme so that procs render nicely */
```

Comment:
I might comment with the exact situation this was fixing (where clauses) in case the rtd theme adjusts in the future to make them display poorly.